### DISCUSSION TITLE
Wing Retention/Removability

### DISCUSSION THREAD

**Discussion by AsaOneal** [robustini/eXploraVTOL#5] [Q&A]
I am currently assembling Explora and was wondering, how are the wings intended to be secured to the fuselage?

**Comment by robustini** [comment]
Hi, if you want join the Discord channel for this kind of question, thanks!
However they connect to the fuselage with four fairly large plastic ties, although we are modifying the fastening system.

**Reply by Salmandakhel** [reply]
Hello, I would like to contact you to develop a drone.

**Reply by robustini** [reply]
> Hello, I would like to contact you to develop a drone.

Hi, contact me on Discord or from my Linkedin profile:

https://www.linkedin.com/in/robustini/

**Comment by Salmandakhel** [comment]
I have added you, and here’s my account:
https://www.linkedin.com/in/salman-al-dakhel%F0%9F%87%B8%F0%9F%87%A6-542303233?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=ios_app


في أربعاء، 12 فبراير، 2025 في 9:51 ص، كتب Marco Robustini <
***@***.***>:

**Reply by robustini** [reply]
> I have added you, and here’s my account: https://www.linkedin.com/in/salman-al-dakhel%F0%9F%87%B8%F0%9F%87%A6-542303233?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=ios_app في أربعاء، 12 فبراير، 2025 في 9:51 ص، كتب Marco Robustini < ***@***.***>:
> […](#)
> Hello, I would like to contact you to develop a drone. Hi, contact me on Discord or from my Linkedin profile: https://www.linkedin.com/in/robustini/ — Reply to this email directly, view it on GitHub <[#5 (reply in thread)](https://github.com/robustini/eXploraVTOL/discussions/5#discussioncomment-12163121)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BPMZ52PIBPDRQ7DXH6ZWZLL2PLVNVAVCNFSM6AAAAABW53R2WKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEMJWGMYTEMI> . You are receiving this because you commented.Message ID: ***@***.***>

I can't connect you via message via Linkedin, you do it then I will reply.
